### PR TITLE
docs: update formatting of user guide

### PR DESCRIPTION
**Pull request by danielleloh**
* edited the formatting to be more consistent
* add white space between features for better readability
* add a table of contents

note: not sure how it will turn out on the website so i'll see what else needs to be edited after merging it :)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -10,26 +10,72 @@ Avengers Assemble (AA) is a **desktop app for managing contacts**, meant for use
...
-3. More advanced commands like filtering and exporting data.
+By following this guide, you will be able to gain a thorough understanding of Avengers Assemble and maximize its potential to streamline your administrative tasks.
+
+<br>
+
+--------------------------------------------------------------------------------------------------------------------
 
 <!-- * Table of Contents -->
-<page-nav-print />
+## Table of Contents
```
Comment (jayllo-c) [inline docs/UserGuide.md:27]: Appreciate the addition of the table of contents here!

**Review by jayllo-c — COMMENTED** [review]
The user guide has been well updated, which makes it more comprehensible to users

**Review by delishad21 — COMMENTED** [review]
Well updated. User guide is now less confusing and more consistent

**Comment by Pughal77** [comment]
Nice! The user guide is now arranged by putting what the user needs more at the top of the page. This definitely improves the usefulness of our guide!

**Pull request merged**